stock_available_mrp/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2014 Numérigraphe SARL, Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Consider the production potential is available to promise",
+    "version": "16.0.1.0.1",
```

Comment:
Please reset to `16.0.1.0.0`

Reply:
ah my bad, i updated `"version": "16.0.1.0.0"`